### ISSUE TITLE
deprecate pkgs.light?

### ISSUE THREAD

**Issue by zackattackz** [NixOS/nixpkgs#354405]
## Issue description

`pkgs.light` had it's source moved to a gitlab mirror recently as per #344482

This is due to the original repo being deleted, because the original author decided to. Furthermore, the program seems to be maintainer-less, unless it is confirmed that the gitlab user hosting the mirror, dpeukert, is going to be doing this.

I'm not sure of nixpkgs' protocol for this situation, but I feel as though unless further information about a maintainer is obtained, the package should be deprecated and removed at some point.

See these sources for more information on the subject:
https://bugs.gentoo.org/937027 - Gentoo bugzilla discussion on the matter.
https://aur.archlinux.org/packages/light?O=10#comment-957514 - original author (supposing this is their account, I'm unsure if it is for sure) confirming the deletion and lack of maintainer.

**Comment by tomodachi94** [comment]
Yeah, that would be concerning. I would drop it if the mirror doesn't plan on maintaining the software. Bonus points if there are alternatives packaged in Nixpkgs.

Pinging maintainers @puffnfresh and @dtzWill for input.

**Comment by Frontear** [comment]
+1 on the drop. I'll also add that I actually also use this app (I've been lazy to move off). The repo had been abandoned for a while even before its deletion, and I don't see anyone who's willing to maintain that source anymore anyways. The sooner we rip the bandaid off the better it would be for everyone who uses the app.

**Comment by zackattackz** [comment]
There is also the `programs.light` nixos setting to consider. I actually only found out about this program because `xbacklight` wasn't working for me, and `programs.light` is the only thing that comes up when you [search for "backlight" on nix options search](https://search.nixos.org/options?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=backlight).

So anyone following a similar path as me is going to start using `light`, and it's probably best they don't start using it unless the situation changes.

**Comment by Frontear** [comment]
Indeed, it might be a good idea to expose new modules for some other (particularly wayland friendly) alternatives. Do we even have modules for other backlight utilities?

**Comment by tomodachi94** [comment]
In related news, [that GitLab mirror](https://gitlab.com/dpeukert/light) has been archived.

**Comment by acid-bong** [comment]
Oh, just found out there's an acpilight module at https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/hardware/acpilight.nix, and its udev rules are even more similar to light's ones:
- [light](https://gitlab.com/dpeukert/light/-/blob/2a54078cbe3814105ee4f565f451b1b5947fbde0/90-backlight.rules#L3) and [acpilight](https://gitlab.com/wavexx/acpilight/-/blob/3f9c62553f4c34c5e21439ae52f6a88155dbcc3c/90-backlight.rules#L6) set both display and keyboard LEDs brightness ownership to `video`
- [brightnessctl](https://gitlab.com/wavexx/acpilight/-/blob/3f9c62553f4c34c5e21439ae52f6a88155dbcc3c/90-backlight.rules#L6) sets display ownership to `video` and keyboard LED -- to `input`

**Comment by xaverdh** [comment]
opened a pr: https://github.com/NixOS/nixpkgs/pull/497125/
